app/services/edge_cache/bust.rb

```diff
@@ -43,73 +34,17 @@ def call
     def determine_provider
       if fastly_enabled?
         "fastly"
-      elsif nginx_enabled? && nginx_available?
+      elsif nginx_enabled?
         "nginx"
       end
     end
 
-    def bust_fastly_cache
-      # TODO: (Alex Smith) - It would be "nice to have" the ability to use the
-      # Fastly gem here instead of custom API calls.
-
-      # @forem/systems Fastly-enabled forems don't need "flexible" domains.
-      HTTParty.post(
-        "https://api.fastly.com/purge/https://#{URL.domain}#{path}",
-        headers: {
-          "Fastly-Key" => ApplicationConfig["FASTLY_API_KEY"]
-        },
-      )
-      HTTParty.post(
-        "https://api.fastly.com/purge/https://#{URL.domain}#{path}?i=i",
-        headers: {
-          "Fastly-Key" => ApplicationConfig["FASTLY_API_KEY"]
-        },
-      )
-    end
-
-    def bust_nginx_cache
-      uri = URI.parse("#{openresty_path}#{path}")
-      http = Net::HTTP.new(uri.host, uri.port)
-      response = http.request Net::HTTP::NginxPurge.new(uri.request_uri)
-
-      raise StandardError, "NginxPurge request failed: #{response.body}" unless response.is_a?(Net::HTTPSuccess)
-
-      response.body
-    end
-
     def fastly_enabled?
```

Comment:
should this be moved inside the provider as well?

Reply:
@rhymes I don't think we can move it, unfortunately. We need to know which provider is enabled before we try to dynamically call on its class, and if we move it, we wouldn't know that 😔